### ISSUE TITLE
Duplicate Album Enteries in Music Library

### ISSUE THREAD

**Issue by kpmelvin** [croneter/PlexKodiConnect#798]
Since version 2.7.5 I’ve been having 2 issues with my Music library and I’m stumped. What happens is after PKC does it’s initial scan, I go into my library in artist view it looks fine, but after I have restarted Kodi and does it’s scheduled scan when I go into artist view and drill into the albums I see duplicate entries for each album. If I go into the duplicate it’s empty.

Near as I can tell when Kodi does it’s scheduled scan I think it’s picking up the “Artists” tag instead of the “Artist” tag causing the second, empty entry. The reason I think this is happening is one of my albums, a classical guitar album the Artist is Andres Segovia but the Artists have several other artists listed. When the Kodi scan finishes scanning this album all the Artists under the Artists tag now show up under the Artist view of the Music library.
[kodi.log](https://github.com/croneter/PlexKodiConnect/files/3015203/kodi.log)

**Comment by croneter** [comment]
You're using direct paths, that's good. For some reason, Kodi keeps scanning your music library - which it should NOT do because Plex supplies everything Kodi needs to know:
```
DEBUG: MUSIC_INFO::CMusicInfoScanner::DoScan Rescanning dir 'smb://192.168.1.101/Music/ACDC/Let There Be Rock/' due to change
ERROR: Could not find 'smb://192.168.1.101/Music/ACDC/Let There Be Rock/01 - Go Down.wav' referenced in cue, case sensitivity issue?
```
I'm looking into solutions right now...

**Comment by kpmelvin** [comment]
I double checked my Kodi configuration. I have Media settings, Library, Music Library, ‘Update library on startup’ set to enabled. Is that OK? I’ve always enabled this in the past without issue.
    	
    	

    	Get Outlook for iOS
    
  




On Thu, Mar 28, 2019 at 5:19 AM -0700, "croneter" <notifications@github.com> wrote:












You're using direct paths, that's good. For some reason, Kodi keeps scanning your music library - which it should NOT do because Plex supplies everything Kodi needs to know:
DEBUG: MUSIC_INFO::CMusicInfoScanner::DoScan Rescanning dir 'smb://192.168.1.101/Music/ACDC/Let There Be Rock/' due to change
ERROR: Could not find 'smb://192.168.1.101/Music/ACDC/Let There Be Rock/01 - Go Down.wav' referenced in cue, case sensitivity issue?



I'm looking into solutions right now...

**Comment by croneter** [comment]
That's fine. It is indeed a PKC bug

**Comment by croneter** [comment]
Should be solved with PKC stable and beta version 2.7.10. Let me know here if this is not the case

# You will need to reset the Kodi database
Go to the PKC settings, then `Advanced`, then `Reset the Kodi database`. Hit YES, then NO
